src/components/TopNav.js

```diff
@@ -3,17 +3,21 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { Link, Redirect } from 'react-router-dom';
 import {
-  Button,
-  Dropdown,
-  Image,
-  Menu,
-} from 'semantic-ui-react';
+  withStyles,
+} from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import Grid from '@material-ui/core/Grid';
+import { Box } from '@material-ui/core';
```

Comment:
Can all of the non-icon imports above be grouped with this one?

Reply:
Yup! I'll fix that. I'm surprised that's not something that eslint can recognize and flag.

Reply:
I prefer one per line like before so it's easier to see changes to single ones